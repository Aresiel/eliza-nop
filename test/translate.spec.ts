import { client } from "./fixtures";

test(`incorrect usage`, async () => {
  const m = await client.sendExpect(
    `eliza japan translate poco loco`,
    /translated.+spanish/i
  );

  expect(m).to.matchSnapshot();
});
