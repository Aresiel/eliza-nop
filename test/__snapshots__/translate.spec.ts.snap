// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`incorrect usage 1`] = `
"translated from Spanish; Castilian to Japanese:
\\"ちょっとおかしい\\""
`;
